model/labels.go

```diff
@@ -97,17 +97,25 @@ var LabelNameRE = regexp.MustCompile("^[a-zA-Z_][a-zA-Z0-9_]*$")
 // therewith.
 type LabelName string
 
-// IsValid is true iff the label name matches the pattern of LabelNameRE. This
-// method, however, does not use LabelNameRE for the check but a much faster
-// hardcoded implementation.
+// IsValid returns true iff name matches the pattern of LabelNameRE
+// for legacy names, and iff it's valid UTF-8 if isUtf8 is true.
```

Comment:
Outdate doc comment? (This should refer to the global `NameValidationScheme` rather than `isUtf8`.)